Dockerfile

```diff
@@ -1,5 +1,22 @@
+# Copyright (c) 2016-present Sonatype, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 FROM       centos:centos7
 MAINTAINER Sonatype <cloud-ops@sonatype.com>
+LABEL vendor=Sonatype \
+  com.sonatype.license="ASLv2" \
```

Comment:
I did com.sonatype.license as that is consistent with how Docker suggests to add labels, particularly to avoid collision (which we were experiencing to some degree since we use centos): https://docs.docker.com/v1.8/userguide/labels-custom-metadata/

docker inspect is one way to view the license information of an image, which will give you much more info. I'm unsure what else is used, if anything.